### PR TITLE
Add read the docs

### PR DESCRIPTION
**Pull request by leo-schick**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
-# Changelog
```
Comment (jankatins) [inline CHANGELOG.md:1]: Sphinx+myst can read md as input

File: LICENSE [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2020 Mara contributors
+Copyright (c) 2020-2022 Mara contributors
```
Comment (jankatins) [inline LICENSE:1]: I think this isn't needed: at a further company the rule was that the year was the year of the creation of the file.

File: README.md [hunk 3]
```diff
@@ -29,237 +29,48 @@ Have a look at a real-world application of Mara Schema in the [Mara Example Proj
  
 &nbsp;
 
-## Defining entities, attributes, metrics & data sets
```
Comment (jankatins) [inline README.md:32]: Personally, I really appreciate readmes with an overview/example to see what the package can do. But links to the docs are the next best thing...

**Comment by leo-schick** [comment]
Available now at https://mara-schema.readthedocs.io/en/latest/

File: README.md (hunk 3 above)
Comment (leo-schick) [reply README.md:32]: You are right. I just couldn't help but did not want to have two places where the documentation is maintained. Do you have a better idea?

File: CHANGELOG.md (hunk 1 above)
Comment (leo-schick) [reply CHANGELOG.md]: now fixed.

**Comment by leo-schick** [comment]
The documentation is mainly based on reStructuredText. This is the same approach I used in `mara-superset`. I am open for discussion if this schould be generally be changed to e.g. MyST but IMO this should then be a architecture decision for all mara packages. I thought already adding a general Mara doc page where we could place such design decisions [like Flask does this](https://flask.palletsprojects.com/en/2.1.x/#additional-notes).

I see this documentation for now as finished.

One thing which should be improved is the [API](https://mara-schema.readthedocs.io/en/latest/api.html) section which looks a bit ugyl. I would like to have it similar to the [Flask API Reference](https://flask.palletsprojects.com/en/2.1.x/api/). I checked the code and it looks like that they build an own package for their Sphinx Theme: [Pallets-Sphinx-Themes](https://pypi.org/project/Pallets-Sphinx-Themes/). For me this was (for now) a level to high so I just kept it for now for mara-superset and for this doc. Maybe someone else can help here.

**Comment by jankatins** [comment]
>  I am open for discussion if this schould be generally be changed to e.g. MyST but IMO this should then be a architecture decision for all mara packages.

My .05€: your work, your decision. I just put that comment in because myst is kind of new, but i guess you already know it :-)

File: README.md (hunk 3 above)
Comment (leo-schick) [reply README.md:32]: I put the context now back again and fixed links for PyPI when using setup.cfg

**Pull request merged**